source/aks/patching-artifactory.html.md.erb

```diff
@@ -1,152 +1,94 @@
 ---
 title: Artifactory Patching
 weight: 30
-last_reviewed_on: 2024-03-13
+last_reviewed_on: 2024-06-06
 review_in: 12 months
 ---
 
 # <%= current_page.data.title %>
 
-Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment. Here's how it is used based on the provided instructions:
+Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment.
 
-The steps below will show you how to check the current version of Artifactory being used (7.71.5) and then to update it to a newer version (7.77.3). This process involves accessing Artifactory's website to determine the latest version available.
+The steps below will show you how to check and update **self-hosted version** of Artifactory being used. This process involves accessing [Artifactory's website](https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes) to determine the latest **self-hosted version** version available.
```

Comment:
```suggestion
The steps below will show you how to check and update **self-hosted version** of Artifactory being used. This process involves accessing [Artifactory's website](https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes) to determine the latest version available.
```

Think this would be fine - version was repeated too